doc/classes/PoolRealArray.xml

```diff
@@ -6,6 +6,7 @@
 	<description>
 		An [Array] specifically designed to hold floating-point values ([float]). Optimized for memory usage, does not fragment the memory.
 		[b]Note:[/b] This type is passed by value and not by reference.
+		[b]Note:[/b] Unlike primitive [float]s which are 64-bit, numbers stored in [PoolRealArray] are 32-bit floats. This means values stored in [PoolRealArray] have lower precision compared to primitive [float]s.
```

Comment:
I'd edit the first line to remove the reference to `[float]` as it can be confusing.